tigerpath/majors_and_certificates/majors/SPO_2018.json

```
@@ -0,0 +1,359 @@
+                    "CRW 5*",
+                    "DAN 3*",
+                    "DAN 4*",
+                    "DAN 5*",
+                    "THR 3*",
+                    "THR 4*",
+                    "THR 5*",
+                    "TRA 3*",
+                    "TRA 4*",
+                    "TRA 5*",
+                    "VIS 3*",
+                    "VIS 4*",
+                    "VIS 5*"
+                  ]
+                }
+              ]
+            }
+          ]
+        },
+        {
+          "name": "Portuguese",
+          "max_counted": 1,
+          "min_needed": "ALL",
+          "explanation": null,
+          "req_list": [
+            {
+              "name": "Prerequisites",
+              "max_counted": 1,
+              "min_needed": 1,
+              "explanation": "All Portuguese concentrators are required to take at least one 200-level POR course.",
+              "course_list": [
+                "POR 2*"
+              ]
+            },
+            {
+              "name": "Upper Division Courses",
+              "max_counted": 1,
+              "min_needed": 5,
+              "explanation": "Any track in the concentration in Spanish and/or Portuguese Literature and Cultures requires a minimum of eight upper-division courses, at least five of which must be in the language of concentration. With the approval of the Director of Undergraduate Studies, up to three courses taken during a semester abroad may count towards the concentration if the student is pursuing Track 1 or Track 2. Up to two courses taken during a semester abroad may be counted towards the concentration if the student is pursuing Track 3 or Track 4.",
+              "course_list": [
+                "POR 2*",
+                "POR 3*",
+                "POR 4*",
+                "POR 5*"
+              ],
+              "excluded_course_list": [
+                "POR 20*"
+              ]
+            },
+            {
+              "name": "Tracks",
+              "max_counted": 1,
+              "min_needed": 1,
+              "explanation": "Departmental courses cover a wide array of literary, cultural, social, historical, and political topics. Students are, therefore, able to pursue courses of study that are tailor-made to their own individual interests. The department offers four different tracks for concentrators.",
+              "req_list": [
+                {
+                  "name": "One Language, Literature, and Culture",
+                  "max_counted": 1,
+                  "min_needed": 3,
+                  "explanation": "Concentration in one language, literature, and cultures (Spanish or Portuguese). Eight upper-division courses in the language of concentration. Up to two of those courses may be taken in English, in which case all written work must be completed in the target language.",
+                  "course_list": [
+                    "POR 2*",
+                    "POR 3*",
+                    "POR 4*"

```

Why not also 500-level here?
```suggestion
                    "POR 4*",
                    "POR 5*"
```